server/core/config/valid/global_cfg.go

```diff
@@ -246,7 +245,7 @@ func (g GlobalCfg) PlatformModeEnabled() bool {
 
 // MergeProjectCfg merges proj and rCfg with the global config to return a
 // final config. It assumes that all configs have been validated.
-func (g GlobalCfg) MergeProjectCfg(log logging.Logger, repoID string, proj Project, rCfg RepoCfg) MergedProjectCfg {
+func (g GlobalCfg) MergeProjectCfg(repoID string, proj Project, rCfg RepoCfg) MergedProjectCfg {
```

Comment:
The `log` field here was unused so I went ahead and removed it for both the legacy and new code dependencies.